stratum/tools/stratum-replay/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Copyright 2020-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM gcc:6.3
```

Comment:
The `gcc` image seems to be a large one (~300-400MB) as it includes the gcc compiler and related tools, which we don't really need at runtime. Can you use a smaller minimal image, e.g., minideb?

Reply:
Didn't noticed that. Will try using the minideb with necessary deps